package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "CapstoneJS",
+  "version": "1.0.0",
+  "description": "Capstone Project",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "build": "webpack",
+    "dev": "webpack --mode development",
+    "start": "webpack serve --open",
+    "deploy": "gh-pages -d dist"
+  },
+  "jest": {
+    "verbose": true
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:DhuruvSingh/CapstoneJS.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/DhuruvSingh/CapstoneJS.git/issues"
+  },
+  "homepage": "https://github.com/DhuruvSingh/CapstoneJS",
+  "devDependencies": {
+    "@babel/core": "^7.18.13",
+    "@babel/preset-env": "^7.18.10",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^29.0.2",
+    "cross-env": "^7.0.3",
+    "css-loader": "^6.7.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-react": "^7.31.7",
+    "html-webpack-plugin": "^5.5.0",
+    "jest": "^29.0.2",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.10.1"
+  },
+  "dependencies": {
```

Comment:
- [ ] While trying to debug the other error, after installing the `file-loader`  you will get to another error saying that  The `loader-utils` module is not available and that means you should install it too 
👇 
![Capture3](https://user-images.githubusercontent.com/90222110/189237475-72756ded-6e20-4baa-8929-c637fad58574.PNG)

Please install it too for your project to run successfully 😉  After installing all of these dependencies I am sure that your project will work fine 👍